iphone/Maps/Maps.xcodeproj/project.pbxproj

```diff
@@ -4689,12 +4728,15 @@
 			baseConfigurationReference = 347526FA1DC0B00F00918CF5 /* common-debug.xcconfig */;
 			buildSettings = {
 				BUNDLE_LOADER = "$(TEST_HOST)";
+				CLANG_ENABLE_MODULES = YES;
```

Comment:
Please try to remove all this autogenerated code from the `buildSettings` and run tests.
It should be OK.

Reply:
+1, this is not needed. We don't use modules.